src/flask_debugtoolbar/__init__.py

```diff
@@ -1,8 +1,17 @@
 import os
 import warnings
 
+import flask
+from packaging import version as version_builder
 from flask import Blueprint, current_app, request, g, send_from_directory, url_for
-from flask.globals import _request_ctx_stack
+
+
```

Comment:
I don't work much with Python these days, but I thought style guides would suggest this as only 1 blank line rather than 2 between the other imports (or even no blank lines since it's just a conditional import and not a distinct function)? Which does `pycodestyle` prefer?

Reply:
Generally `black` prefers 2 lines between functions but there's no set rule for spaces between imports and your first expressions. This is also a fun one because it's technically handling an import but it's also an expression.

Do you have a preference? I don't have `pycodestyle` installed locally.

Reply:
I'm cool either way!